cfgov/unprocessed/css/organisms/item-introduction.less

```diff
@@ -35,11 +35,28 @@
 */
 
 .o-item-introduction {
+    // visually approximates 60px
+    margin-bottom: unit( 55px / @base-font-size-px, em);
+
     .short-desc {
-        padding-bottom: unit( @grid_gutter-width / 2 / @base-font-size-px, em)
+        padding-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
```

Comment:
I don't think parens are necessary but I could be wrong. I think the order of operations takes care of the precedence.